config/schemas/settings.json

```diff
@@ -363,12 +363,12 @@
 			"type" : "object",
 			"default": {},
 			"additionalProperties" : false,
-			"required" : [ "repositoryURL", "enableInstalledMods", "autoCheckRepositories" ],
+			"required" : [ "repositoryURL", "enableInstalledMods", "autoCheckRepositories", "updateOnStartup", "updateConfigUrl" ],
 			"properties" : {
 				"repositoryURL" : {
 					"type" : "array",
 					"default" : [
-						"http://download.vcmi.eu/mods/repository/repository.json"
+						"https://raw.githubusercontent.com/Nordsoft91/vcmi-mods-repository/mods-from-github/repository.json"
```

Comment:
Link will be corrected after merging https://github.com/vcmi/vcmi-mods-repository/pull/1